scripts/UPDATE.sh

```diff
@@ -30,6 +30,14 @@
 set -e
 # give a sensible error message (note: works only in bash)
 trap 'echo An error occurred in $0 at line $LINENO. Current working-dir: $PWD' ERR
+SIRF_TAG='default'
+while getopts :t: option
+ do
+ case "${option}"
+  in
+  t) SIRF_TAG=$OPTARG;;
```

Comment:
add a "fallthrough" and help. something like
```bash
h)
   echo "Usage: $0 [-t tag]"
   echo "Use the tag option to checkout a specific version."
   echo "Otherwise the most recent release will be used."
   exit 
   ;;
*)
   echo "Wrong option passed."
   exit 1
  ;;
```